modules/network_synchronizer/networked_controller.cpp

```diff
@@ -1181,7 +1182,25 @@ void DollController::receive_inputs(Vector<uint8_t> p_data) {
 }
 
 int DollController::calculates_sub_ticks(real_t p_delta, real_t p_iteration_per_seconds) {
-	return 1; // TODO
+	// TODO use the same algorithm the server uses to determinates the speedup
+	// so the input pool is enlarged when interned problems are detected and constrained
```

Comment:
Did you mean "internal problems"?

Reply:
Oh, that message is just a quick reminder, bad written. It's about internet problems, like packets lost, latency, fluctuations, etc...

Reply:
No worries that's fine, I'm just trying to stay up to date on stuff
Keep up the good work!